gotocompany/compass/v1beta1/service.proto

```diff
@@ -1198,6 +1212,12 @@ message CreateAssetProbeResponse {
   string id = 1;
 }
 
+message SyncAssetsRequest {}
+
+message SyncAssetsResponse {
+  uint32 num_affected = 1;
```

Comment:
are we planning to have immediate result when calling the API? it the data is huge, it will take a long time and lead to timeout